tests/operator_test.go

```diff
@@ -1616,7 +1616,7 @@ spec:
 			allPodsAreReady(kv)
 		})
 
-		It("[QUARANTINE][sig-compute][test_id:3150]should be able to update kubevirt install with custom image tag", func() {
```

Comment:
If you just have a suspicion it is due to short timeouts, would it make sense to keep it in quarantine and monitor its stability on periodics?

Reply:
yes, I think it would be better